scientific/src/commonMain/kotlin/converter/area/convertToKinematicViscosity.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ Copyright 2021 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.converter.area
+
+import com.splendo.kaluga.scientific.PhysicalQuantity
+import com.splendo.kaluga.scientific.ScientificValue
+import com.splendo.kaluga.scientific.converter.kinematicViscosity.kinematicViscosity
+import com.splendo.kaluga.scientific.unit.Area
+import com.splendo.kaluga.scientific.unit.Time
+import com.splendo.kaluga.scientific.unit.per
+import kotlin.jvm.JvmName
+
+@JvmName("areaDivTime")
```

Comment:
Here however, you should specify special metric and imperial cases. The reason is that `unit per time.unit`, not knowing the unit of the area, will use its default:

```
infix fun MetricArea.per(time: Time) = MetricKinematicViscosity(this, time)
infix fun ImperialArea.per(time: Time) = ImperialKinematicViscosity(this, time)
infix fun Area.per(time: Time): KinematicViscosity = when (this) {
    is MetricArea -> this per time
    is ImperialArea -> this per time
}
```
Ergo, you'll get KinematicViscosity instead of MetricKinematicViscosity or ImperialKinematicViscosity here.

Reply:
nice catch, i totally missed that.